pywinauto/controls/hwndwrapper.py

```diff
@@ -737,8 +735,8 @@ def wait_for_idle(self):
         win32functions.WaitGuiThreadIdle(self.handle)
 
     # -----------------------------------------------------------
-    def click(
-        self, button = "left", pressed = "", coords = (0, 0), double = False, absolute = False):
+    def click(self, button = "left", pressed = "", coords = (0, 0),
+              double = False, absolute = False):
         """
```

Comment:
PEP-8 recommends no spaces in parameters default values. https://www.python.org/dev/peps/pep-0008/#other-recommendations